modules/nf-core/ltrharvest/tests/main.nf.test

```diff
@@ -0,0 +1,60 @@
+nextflow_process {
+
+    name "Test Process LTRHARVEST"
+    script "../main.nf"
+    process "LTRHARVEST"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ltrharvest"
+
+    test("homo_sapiens-genome_21_fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.gff3).match("gff3") },
+                { assert path(process.out.scn[0][1]).text.contains("46510803 46520182 9380 46510803 46510940 138 46520042 46520182 141 86.52 0 chr21") },
+                { assert snapshot(path(process.out.versions[0]).text).match("script_versions") }
+            )
+        }
+
+    }
+
+    test("homo_sapiens-genome_fasta-stub") {
+
+        options '-stub'
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() },
+                { assert snapshot(path(process.out.versions[0]).text).match("stub_versions") }
```

Comment:
the `stub_versions` and `script_versions` snapshots look the same. I think you can share those snapshots between test cases (`.match("versions")`) since the script and stub blocks use the exact same logic

Reply:
it would also be ok to remove this check entirely or loosen it a bit to just check for existence, so you don't need to update the snapshots for minor version bumps... but its fine if you want to be conservative here

Reply:
Hi @ahvigil 

Thank you for the feedback. It is not possible anymore to use the same snapshot name. nf-test doesn't allow that anymore.

Reply:
ugh thats too bad but it seems you're right: https://github.com/askimed/nf-test/pull/188

however, there is an open issue to provide an opt in for cases like this where it might be useful to remove repetition: https://github.com/askimed/nf-test/issues/197. Hope they can implement something like that!